src/DriverManager.php

```diff
@@ -96,7 +96,8 @@ final class DriverManager
      *
      * @deprecated Use actual driver names instead.
      *
-     * @var string[]
+     * @var array<string, string>
```

Comment:
Is this line redundant when `@psalm-var` is used? We don't generate docs from PHPDoc docblocks and the psalm annotation does the trick too.

Reply:
I supposed that native PHPDoc tags were kept in order to maintain compatibility with tools (IDEs, etc) that don't support Psalm extended typing.
If you think they are not required, I'm fine removing them.

Reply:
I guess you have a point that some IDEs won't understand the psalm part. :+1: